### ISSUE TITLE
Possibility to decrypt without mel-cp?

### ISSUE THREAD

**Issue by KarolisKirna** [meld-cp/obsidian-encrypt#22]
Would it be possible to decrypt the text without meld-cp plugin. My worry is if you lose interest in maintaining the plugin what will happen with my encrypted data :)

**Comment by ccchan234** [comment]
somehow a standalone rescue boat like a standalone decryptor will help if possible, thanks

**Comment by meld-cp** [comment]
I agree, I started working on [A standalone decrypt tool](https://github.com/meld-cp/obsidian-encrypt/blob/main/src/tools/decrypt.html)

It's in it's early stages and there's no mention of it on the main pages yet.

The idea is to simply save the file locally and open it in your browser for offline decryption if needed.

Still some work to do here though.
